web-wallet/src/routes/components-showcase/Toasts.svelte

```diff
@@ -2,26 +2,25 @@
   import { Button, Toast } from "$lib/dusk/components";
   import { toast } from "$lib/dusk/components/Toast/store";
   import { mdiCopyleft } from "@mdi/js";
+
+  function triggerToast() {
+    toast("success", "Toast message", mdiCopyleft);
+  }
 </script>
 
 <section style="flex-direction:column">
   <Button
     text="toggle toast"
     variant="secondary"
     size="small"
-    on:click={() => toast("success", "Toast message", mdiCopyleft)}
+    on:click={triggerToast}
   />
 
   <Button
     text="toggle another toast"
     variant="secondary"
     size="small"
-    on:click={() =>
```

Comment:
This way you're removing a test case (two actually: error + text wrapping).

Reply:
I didn't even see it, have added it back.

Generally, I wouldn't expect the components showcase to be used to cover different test scenarios, although I agree it's useful to have more variations of a component.

Reply:
Well, it doesn't replace a storybook, obviously, but it has been useful to quickly check the styles: so variations are welcome.